ReadMe.md

```diff
@@ -1,4 +1,4 @@
-# DITA Open Toolkit Plug-in Registry
+# DITA Open Toolkit Plug-in Registry [![Netlify Status](https://api.netlify.com/api/v1/badges/626192a4-0b63-4a52-9b5a-ef6b46dc6464/deploy-status)](https://app.netlify.com/sites/gracious-snyder-3a4d31/deploys)
```

Comment:
I would prefer to rename the autogenerated Netlify site to use something more indicative of its actual purpose.

Reply:
Sure, we will be using custom plugins.dita-ot.org for this in the end.

Reply:
Renamed to plugins-dita-ot